partner_employee_quantity/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================
+Employee quantity in partners
+=============================
+
+This module allows to set employee quantity for partners, and categorize them
+by ranges.
+
+Configuration
+=============
+
+To manage available ranges, you need to:
+
+* Go to *Sales > Configuration > Address Book > Ranges by quantity of employees*.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to any partner's form.
+* Enable *Is company?*.
+* Go to *Sales & Purchases > Employee quantity*.
```

Comment:
This information is not related with sales & purchase, it must be shown in the main form.